### PR TITLE
Fix ci-test-tasks, add patchTaskVersion

### PR DESCRIPTION
**Pull request by PavloAndriiesh**
**Task name**: <Name of changed or new pipeline task>

**Description**: <Describe your changes here>

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Comment by jmyersmsft** [comment]
@PavloAndriiesh What are you doing with this PR? Every time you push to your branch you're sending emails to the entire Azure Artifacts Packaging team

File: ci/patch-task-version.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  patch: params.version.split('.')[2],
+}
+
+console.log(version);
+
+// content.version.Major = version.major;
+content.version.Minor = version.minor;
+content.version.Patch = version.patch;
+
+fs.writeFileSync(pathToJson, JSON.stringify(content));
```
Comment (mpodriezov) [inline ci/patch-task-version.js:30]: end of file

**Review by mpodriezov — APPROVED** [review]

File: ci/patch-task-version.js (hunk 1 above)
Comment (PavloAndriiesh) [reply ci/patch-task-version.js]: fixed

**Pull request merged**